tasks/transifex-resjson-tasks.js

```diff
@@ -720,7 +720,7 @@ module.exports = function (grunt) {
         undefined if the code isn't of correct format.
     */
     function mapToTxLangCode(str) {
-        var matcher = str.match(/([a-z]{2})-([A-Z]{2}|latn)$/i);
+        var matcher = str.match(/([a-z]{2,3})(?:[-_]([a-z]{2}|latn))?$/i);
```

Comment:
Why include the underscore (`_`) as accepted separator in the regexp? Otherwise this looks like a easy way to improve the mappings quickly.